doc/permssions.md

```diff
@@ -0,0 +1,14 @@
+# Minimum permissions for collector account
+
+The **collector** only gathers information from vCenter, therefore a **read only** permission whould be enough. 
+
+One such role is the [Read Only Role](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-93B962A7-93FA-4E96-B68F-AE66D3D6C663.html).
+
+This permissions needs to be applied at the root object level with "Propagate to children" selected.
+
+> Remark: The **Read Only** role is not sufficient for MTV operator.
+
+## Credentials to login into vCenter
+
+In order for the **collector** to be able to login into vCenter, `username` and `password` must be supplied. 
```

Comment:
To enable the **collector** (or another name we will decide) to log in to vCenter, the `username` and `password` must be provided.